pontoon/base/views.py

```diff
@@ -427,6 +433,106 @@ def entities(request):
     }, safe=False)
 
 
+@login_required(redirect_field_name='', login_url='/403')
+@require_POST
+@require_AJAX
+@transaction.atomic
+def batch_edit_translations(request):
```

Comment:
I'll see what I can do with stats, celery is also an option, but probably not the most user friendly one.